backend/src/test/java/ca/gov/dtsstn/cdcp/api/web/v1/model/mapper/SubscriptionModelMapperTest.java

```diff
@@ -0,0 +1,34 @@
+package ca.gov.dtsstn.cdcp.api.web.v1.model.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mapstruct.factory.Mappers;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import ca.gov.dtsstn.cdcp.api.service.domain.Subscription;
+import ca.gov.dtsstn.cdcp.api.service.domain.ImmutableSubscription;
+
+@ExtendWith({ MockitoExtension.class })
+class SubscriptionModelMapperTest {
+
+	SubscriptionModelMapper subscriptionModelMapper = Mappers.getMapper(SubscriptionModelMapper.class);
+
+	@Test
+	final void testToModelStringIterableOfSubscriptionNullSubscription() {
```

Comment:
You should add a `@DisplayName()` annotation here

Reply:
Implemented in commit commit [9fbe7e7](https://github.com/DTS-STN/canadian-dental-care-plan/pull/1865/commits/9fbe7e70e363ba40945580d5841081219b4bf781).